Sharing/ViewModels/ViewModel.swift

```diff
@@ -153,20 +159,33 @@ final class ViewModel: ObservableObject {
     ///   - contact: Contact to share.
     ///   - completionHandler: Handler to process a `success` or `failure` result.
     func createShare(contact: Contact, completionHandler: @escaping (Result<(CKShare, CKContainer), Error>) -> Void) {
```

Comment:
The documentation and function name are no longer correct. If we agree that this PR solves a problem we can change this by either changing the name & documentation, or creating a separate function that calls this one. Something like this:

```
func share(… // This function checks if `contact.associatedRecord.share != nil` and based on that uses one of the two private functions below

private func createShare(…

private func getExistingShare(…
```

Reply:
Fixed in https://github.com/apple/cloudkit-sample-sharing/pull/6/commits/afef4791f76d59a263194b3efc9c24825683165f